.vib/kong/goss/vars.yaml

```diff
@@ -8,10 +8,6 @@ directories:
       - /opt/bitnami/kong/conf
   - paths:
       - /docker-entrypoint-initdb.d
-      - /opt/bitnami/kong/openresty/luajit/share/lua/5.1/kong/include/opentelemetry
+      - /opt/bitnami/kong/include/opentelemetry
       - /opt/bitnami/kong/bin
-      - /opt/bitnami/kong/server/lib
-files:
-  - paths:
-      - /opt/bitnami/kong/server/lib/pluginsocket.proto
```

Comment:
Is the `../kong/server/lib` folder no longer present? Is this related to the Lua deprecation as well?

Reply:
It does not exist anymore:
```bash
root@04b2c80004a5:/# ls /opt/bitnami/kong/server/
root@04b2c80004a5:/# 
```